src/lib/utils/functions/EntityTypeUtils.ts

```diff
@@ -162,3 +198,23 @@ export function hasFilesInView(entityView: EntityView) {
 export function isFileView(entityView: EntityView) {
   return entityView.viewTypeMask === ENTITY_VIEW_TYPE_MASK_FILE
 }
+
+export function isVersionableEntity(
+  entity: Entity,
+): entity is VersionableEntity {
+  return isVersionableEntityType(convertToEntityType(entity.concreteType))
+}
+
+export function getVersionDisplay(entity: Entity): string {
+  if (!isVersionableEntity(entity)) {
+    console.warn("Entity isn't versionable:", entity)
+    return ''
+  }
+  if (!entity.isLatestVersion || !isTable(entity)) {
+    return entity.versionNumber!.toString()
```

Comment:
Is this the expected behavior?  If this is the current version of a file, they want it to show the version number rather than "Current"?

Reply:
It looks like what we do today for files is `${versionNumber} (Current)`, so I'll update to match